_includes/manuals/2.0/1.2_release_notes.md

```diff
@@ -442,6 +442,7 @@ External authentication sources can now be assigned to taxonomies using a new UI
 
 ### Upgrade warnings
 
+* The CentOS packages are not tested on Scientific Linux or Oracle Linux anymore. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.
```

Comment:
```suggestion
* The CentOS packages are not tested on Scientific Linux or Oracle Linux anymore. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.
```

anymore implies we ever did but AFAIK this was always best effort. I'd say we're formalizing this and changing its status from supported to best-effort.